Database connection using active records

Sboursen opened the pull request:
# Description

Milestone #3

- [x] Created the necessary migration files.
- [x] Foreign keys are included.
- [x] All columns that are foreign keys have a corresponding index.

Cheers,

Sboursen committed:
Add ERD to the docs

Sboursen committed:
Update the readme

Sboursen committed:
Add CreateUsers migration

Sboursen committed:
Remove updated_at and created_at as they are generated automatically

Sboursen committed:
Add migration for post

Sboursen committed:
Add migration for comment

Sboursen committed:
Add migration for likes

Sboursen committed:
Change the foreignkey table name

Sboursen committed:
Add index to foreign keys

uwadonat commented on db/migrate/20220603095337_create_posts.rb:
```diff
@@ -0,0 +1,11 @@
+  def change
+    create_table :posts do |t|
+      t.string      :title
+      t.text        :text
+      t.integer     :comments_count, default: 0
+      t.references :user, foreign_key: true
```
- [x] Let's add `likes_counter` in this migration to match [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

Sboursen replied:
@uwadonat 
Good catch, totally missed that.
Cheers,

uwadonat replied:
Nice!

uwadonat reviewed (changes requested):
Hi @Sboursen 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit and pull request messages are very descriptive ✔️
- No linter error ✔️
- Your code is well organized and clean ✔️
![almost](https://user-images.githubusercontent.com/65860678/171847434-b7f7e1b1-8c66-40f5-9f0b-e18e93b017d8.gif)

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sboursen committed:
Make changes

- Remove the `Add index` migration table.
- Add `likes_counter` column in the `posts` table

uwadonat reviewed (approved):
## Hi @Sboursen 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

![cong](https://user-images.githubusercontent.com/65860678/171854111-221efdc4-26a9-4b3e-96fb-92ffba1aba8f.jpg)

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
